styles/bahmniAppsFormBuilder/_form.scss

```diff
@@ -359,6 +363,12 @@ $lightestGray: #eee;
           }
         }
       }
+      .hidden {
+        display: block !important;
```

Comment:
Yes, it is usually not preferred. But the element here actually has lot of css overriding happening (if we change any of those, the impact is more and in different places). So to forcefully make this happen and to avoid impact on remaining places, we are using `!important`.